wcivf/apps/people/models.py

```diff
@@ -153,6 +153,76 @@ def get_max_facebook_ad_spend(self):
             )
         )
 
+    @property
+    def facebook_personal_username(self):
+        facebook_personal_url = self.facebook_personal_url
+        facebook_split = list(filter(None, facebook_personal_url.split("/")))
+        facebook_personal_username = facebook_split[-1]
+
+        return facebook_personal_username
+
+    @property
+    def facebook_username(self):
+        facebook_page_url = self.facebook_page_url
+        facebook_split = list(filter(None, facebook_page_url.split("/")))
+        facebook_username = facebook_split[-1]
```

Comment:
Minor thing, but when you set a variable and then return it immediately without doing anything else, you can simply return instead:
```
@property
def facebook_username(self):
    facebook_page_url = self.facebook_page_url
    facebook_split = list(filter(None, facebook_page_url.split("/")))
    return facebook_split[-1]
```